src/oemof/solph/_exceptions.py

```diff
@@ -0,0 +1,6 @@
+class FlowOptionWarning(UserWarning):
+    pass
+
+
+class WrongOptionCombinationError(ValueError):
+    pass
```

Comment:
We have the other warnings in the tools package, to allow reusing them. It might make sense to do this for these exceptions as well.

(Just as a point for discussion, not a reason to reject this PR.)

Reply:
I only saw one other warning `SuspiciousUsageWarning` in `oemof.tools`. We could define them there if this is needed. I personally find a bit cumbersome to have to create 2 PRs in 2 different packages, but I will abide :)